gratipay/models/package/__init__.py

```diff
@@ -40,3 +51,35 @@ def from_names(cls, package_manager, name):
         """
         return cls.db.one("SELECT packages.*::packages FROM packages "
                           "WHERE package_manager=%s and name=%s", (package_manager, name))
+
+
+    @property
+    def team(self):
+        """A computed attribute, the :py:class:`~gratipay.models.team.Team`
+        linked to this package if there is one, otherwise ``None``. Makes a
+        database call.
+        """
+        return self._load_team(self.db)
+
+
+    def ensure_team(self, cursor, owner):
+        """Given a db cursor and :py:class:`Participant`, insert into ``teams`` if need be.
+        """
+        team = self._load_team(cursor)
```

Comment:
Alright, so there's work to do. I think I had a notion that a db constraint would cause an `IntegrityError` in subsequent parallel calls, but clearly that's false. Let's see here ... :thinking: